src/common/arrays_input.c

```diff
@@ -30,12 +30,30 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
 #include "c_common/time_msg.h"
 #include "c_common/debug_macro.h"
-
+/** 
+ * @brief Function for array input
+ * @details This function generates the array inputs according to their type 
+ * received through @a ArrayType *v parameter and store them in @a c_array. It
+ * can be empty also if received @a allow_empty true. The cases of failure are:-
+ * 1. When @a ndim is not equal to one dimension. 
+ * 2. When no element is found i.e. nitems is zero or negative. 
+ * 3. If the element type doesn't lie in switch cases, give the error of expected array of any integer type
+ * 4. When size of @a c_array is out of range or memory. 
+ * 5. When null value is found in the array. 
+ * 
+ * All these failures are represented as error through @a elog.
+ * @param *v The type of element to be processed.
```

Comment:
`@param` should have the name of the parameter and `*v` is not the name: `v` is the name.

Reply:
You should also indicate if it is an [in], [out], or [in,out] parameter